azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -493,8 +520,14 @@ public void start() {
         this.executionCleanupIntervalMin, TimeUnit.MINUTES);
     this.cleanupService.scheduleAtFixedRate(this::cleanUpContainersInTerminalStatuses, 0L,
         this.containerCleanupIntervalMin, TimeUnit.MINUTES);
-    this.cleanupService.scheduleAtFixedRate(this::cleanUpDanglingYarnApplications, 0L,
-        this.yarnAppCleanupIntervalMin, TimeUnit.MINUTES);
+    if (this.yarnAppCleanupIntervalMin <= 0) {
```

Comment:
Do we need <= 0 check? I think if we want to disable this feature, simply leave yarnAppCleanupIntervalMin==0 is enough

Reply:
yes while it's of no harm and tolerant more ;)